Wpf/Microsoft.DataTransfer.WpfHost/App.xaml

```diff
@@ -29,10 +29,10 @@
       <system:String x:Key="StartNewImportKey">New Import</system:String>
 
       <!-- Step: Welcome -->
-      <system:String x:Key="WelcomePageTitleKey">DocumentDB Data Migration Tool</system:String>
+      <system:String x:Key="WelcomePageTitleKey">Azure Cosmos DB Data Migration Tool</system:String>
       <Span x:Key="WelcomePageContentKey">
         <Span>
-          Use the DocumentDB data migration tool to import data to DocumentDB from a variety of sources.
+          Use the Azure Cosmos DB data migration tool to import data to Cosmos from a variety of sources.
```

Comment:
import data to "Cosmos DB".

Reply:
I changed the name from ``Cosmos`` to ``Cosmos DB``